staging/src/k8s.io/apiserver/pkg/endpoints/handlers/rest.go

```diff
@@ -416,3 +418,19 @@ func parseTimeout(str string) time.Duration {
 func isDryRun(url *url.URL) bool {
 	return len(url.Query()["dryRun"]) != 0
 }
+
+type grpcError interface {
+	Code() grpccodes.Code
+	Error() string
+}
+
+func isTooLargeError(err error) bool {
```

Comment:
we could also have the storage layer expose this, to avoid depending on grpc codes here